app/src/main/java/org/oppia/app/utility/FontScaleConfigurationUtil.kt

```diff
@@ -0,0 +1,33 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.content.Context.WINDOW_SERVICE
+import android.util.DisplayMetrics
+import android.view.WindowManager
+import org.oppia.app.model.StoryTextSize
+
+object FontScaleConfigurationUtil {
```

Comment:
Please add a KDoc explaining what this class is. Also, can we name this to be more specific than just a 'util'? E.g. maybe 'DisplayFontScaler'?

Reply:
Will be adding Kdocs after finalising the approach.